content/about/faq.md

```diff
@@ -45,7 +45,7 @@ menu: "footer"
 
 * <a name="faq-moduleorder"></a>**How do I change the order in which modules are applied to an image?**<a href="#faq-moduleorder" class="anchor" title="Link to this FAQ entry">¶</a>
 
-    See the [user manual](https://docs.darktable.org/usermanual/3.8/en/darkroom/pixelpipe/the-pixelpipe-and-module-order/#changing-module-order) for more details.
+    See the [user manual](/usermanual/en/darkroom/pixelpipe/the-pixelpipe-and-module-order/#changing-module-order) for more details.
```

Comment:
Note that the user manual is on a subdomain (**docs**.darktable.org) so this won't work. Also you can't remove the version number from the URL since this will lead to a 404 error, though we could put `/stable/` instead.

Reply:
> Note that the user manual is on a subdomain (docs.darktable.org) so this won't work.

I believe this does work, because:
- the relative link will work out to `www.darktable.org/usermanual`
- there is a redirect from `www.darktable.org/usermanual` (without the `/stable/` or any version) to `docs.darktable.org/usermanual/stable`
- then `docs.darktable.org/usermanual/stable` redirects to the current stable version

I believe that this following link should demonstrate this: https://www.darktable.org/usermanual/en/darkroom/pixelpipe/the-pixelpipe-and-module-order/#changing-module-order. Whether this is the best way to set up the links (vs. pinning versions explicitly, or linking directly to `docs.darktable.org` URLs), I'm not sure -- for now I'm following the pattern used for other links to the manual in this FAQ to be consistent.

Reply:
Right ok I understand now. The previous version of the user manual used to be on darktable.org/usermanual. That redirect was to pick up people using old links. I think I would prefer to change all of the _other_ links in the FAQ to point to the new (stable) location since this is the URL we want to advertise -- i.e. have them all point to docs.darktable.org/usermanual/stable.

Reply:
Done. As noted in the new PR description, I also had to update a couple of links because the existing links were broken.